Automattic/wp-calypso
# My Home: Move the secondary location into its own component

#### Changes proposed in this Pull Request

This PR moves the secondary location of My Home (the right column) to its own component, in preparation for cards being displayed according to server-side logic. This finalizes the work started in #40368. No visual or behavioral changes should occur.

<img width="976" alt="Screen Shot 2020-03-27 at 14 58 44" src="https://user-images.githubusercontent.com/1233880/77764514-0ce6f500-703d-11ea-80f0-943eafd13447.png">

This has allowed to simplify the main Home component into a functional component now that all locations are in separated components. Some unused styles have been cleaned up too.

#### Testing instructions

- Load this branch, and go to `/home/:site`.
- Verify it visually looks the same in production (load the same route on wordpress.com).
